content/en/cloud_cost_management/custom_allocation_rules/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Custom Allocation Rules
+description: "Allocate cloud costs based on custom allocation rules."
+further_reading:
+- link: "https://docs.datadoghq.com/cloud_cost_management/?tab=aws#overview"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+---
+
+## Overview
+
+Custom cost allocation allows you to showback or chargeback your costs by attributing selected costs to relevant business dimensions. Once you have set up allocation rules, you can report on which costs were allocated by a rule. 
+
+Custom cost allocation follows Tag Pipelines, enabling allocations based on user-defined tags. Costs are allocated on a daily basis, and can be applied to Cloud Cost metrics from AWS, GCP, and Azure. 
+
+## Rule creation
+
+Access the Custom Allocation Rules section under [Cloud Cost settings][1] to establish rules. Choose from the following allocation methods:
```

Comment:
```suggestion
Access the [Custom Allocation Rules section][1] under Cloud Cost settings to establish rules. Choose from the following allocation methods:
```

Reply:
Suggesting this update since the link leads directly to the Custom Allocation Rules section